substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1266,13 +1266,13 @@ mod remote_tests {
 	use std::os::unix::fs::MetadataExt;
 	use std::env;
 
-	static TRANSPORT_URL: once_cell::sync::Lazy<String> = once_cell::sync::Lazy::new(|| {
-        env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string())
-    });
+	fn endpoint() -> String {
+		env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string())
+	}
 
 	#[test]
 	fn print_transport_url() {
-		println!("TRANSPORT_URL: {:?}", once_cell::sync::Lazy::<std::string::String>::force(&TRANSPORT_URL));
+		println!("TRANSPORT_URL: {:?}", endpoint());
 	}
```

Comment:
Just's remove this test and put a debug log instead in `fn endpoint` I think

Reply:
If I do smth like this:
```rust
fn endpoint() -> String {
	let endpoint_value = env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string());
	println!("Endpoint: {:?}", endpoint_value);
	endpoint_value
}
```
I end up with an output like this:
```
running 13 tests
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
✅ Found 9 keys (1.75s)
✅ Found 0 keys (1.69s)
test remote_tests::can_build_big_pallet ... ok
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
✅ Found 1 keys (1.70s)
✅ Found 641 keys (2.00s)
[00:00:00] ✅ Downloaded key values 35.9984/s [====================================================================================] 9/9 (0s)
[00:00:00] ✅ Downloaded key values 4.1249/s [=====================================================================================] 1/1 (0s)
✅ Found 641 keys (2.14s)
✅ Found 641 keys (2.25s)
✅ Found 641 keys (2.19s)
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ 0/641
░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ 0/641
✅ Inserted keys into DB (0.00s)
✅ Found 641 keys (1.99s)
✅ Inserted keys into DB (0.00s)
✅ Found 641 keys (2.00s)
✅ Found 1 keys (1.06s)
[00:00:00] ✅ Downloaded key values 4.0373/s [=====================================================================================] 1/1 (0s)
✅ Inserted keys into DB (0.00s)
[00:00:02] Downloading key values 156.1834/s [==============================================>----------------------------------] 366/641 (2s)
test remote_tests::can_create_child_snapshot ... ok
✅ Loaded snapshot (0.01s)
✅ Loaded snapshot (0.00s)
test tests::can_exclude_from_snapshot ... ok
✅ Loaded snapshot (0.00s)
test tests::can_load_state_snapshot ... ok
[00:00:02] ✅ Downloaded key values 214.2809/s [===============================================================================] 641/641 (0s)
✅ Inserted keys into DB (0.02s)
✅ Loaded snapshot (0.00s)
test remote_tests::snapshot_block_hash_works ... ok
[00:00:02] ✅ Downloaded key values 214.4463/s [===============================================================================] 641/641 (0s)
[00:00:03] ✅ Downloaded key values 211.0836/s [===============================================================================] 641/641 (0s)
✅ Inserted keys into DB (0.01s)
[00:00:03] ✅ Downloaded key values 213.1888/s [===============================================================================] 641/641 (0s)
⠋     0.000 s   Inserting keys into DB...test remote_tests::can_create_snapshot ... ok
[00:00:03] ✅ Downloaded key values 206.8499/s [===============================================================================] 641/641 (0s)
✅ Inserted keys into DB (0.01s)
test remote_tests::can_build_one_small_pallet ... ok
✅ Inserted keys into DB (0.01s)
✅ Inserted keys into DB (0.01s)
✅ Found 641 keys (2.01s)
✅ Found 1 keys (0.99s)
[00:00:04] ✅ Downloaded key values 155.4818/s [===============================================================================] 641/641 (0s)
✅ Inserted keys into DB (0.01s)
[00:00:00] ✅ Downloaded key values 4.0076/s [=====================================================================================] 1/1 (0s)
✅ Loaded snapshot (0.00s)
test remote_tests::offline_else_online_works ... ok
✅ Inserted keys into DB (0.00s)
⠙     1.759 s   Scraping keys...█████████████████████████████████████████████████████████████████████████████████████████████████████████████
Endpoint: "wss://rococo-try-runtime-node.parity-chains.parity.io:443"
✅ Found 495 keys (1.81s)
```
And if I use `debug!("Endpoint: {}", endpoint_value);` instead, then I see no output from that statement in the terminal.

Reply:
let's just set the default to something sensible and leave the log out

Reply:
Removed.